glitch/tests/security/docker/files/weak_crypt.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu
+USER ubuntu
+
+RUN md5sum foo
```

Comment:
This test actually shows something that we probably should change 😅 We are considering the keyword "checksum" but not "sum" or "md5sum". Could you create an issue with this problem and change this test please?

Reply:
You're refering to the checksum list in the config files? If so, yes I can create the issue but what do you want to change in the test, I didn't understand

Reply:
Yes, the list should be changed. I was thinking the test should be changed to a situation where MD5 (for example) is used in an unsafe context. For checksums, MD5 is fine. To encrypt sensitive information, MD5 should not be used.

Reply:
Changed the test to be used with sensitive information